sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1693,6 +1764,28 @@ export { deserializationPolicy }
 // @public
 export type EncryptionAlgorithmType = 'AES256';
 
+// @public
+export interface FilterBlobItem {
```

Comment:
`FilteredBlobItem`?

Reply:
This is a generated type `FilteredBlobItem`, align cross lanauges. So we can leave it is instead do manual wrap for the type in JS only.